### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by EPashkin**
Apply https://github.com/gtk-rs/gir/pull/424, https://github.com/gtk-rs/gir/pull/426
and fix\ignore clippy warnings

### CODE REVIEW

File: src/source.rs [hunk 1]
```diff
@@ -63,6 +63,12 @@ impl CallbackGuard {
     }
 }
 
+impl Default for CallbackGuard {
```
Comment (EPashkin) [inline src/source.rs]: Not sure that adding implementation better that ignoring warning for this case.

**Comment by sdroege** [comment]
Looks good to me :+1:

**Comment by GuillaumeGomez** [comment]
To me too. Thanks!

**Comment by EPashkin** [comment]
Only last https://github.com/gtk-rs/gdk/pull/184, https://github.com/gtk-rs/pango/pull/89 ;)

**Pull request merged**